functions/Invoke-DbaLogShipping.ps1

```diff
@@ -664,7 +664,7 @@ The script will show a message that the copy destination has not been supplied a
 		}
 
 		# Check the backup network path
-		if (-not ((Test-Path $BackupNetworkPath -PathType Container -IsValid -Credential $SourceCredential) -and (Get-Item $BackupNetworkPath).PSProvider.Name -eq 'FileSystem')) {
+		if ((Test-DbaSqlPath -Path $BackupNetworkPath -SqlInstance $SourceSqlInstance -SqlCredential $SourceCredential) -ne $true){
```

Comment:
This changes the intention of this check the way I am looking at it.

`Test-DbaSqlPath` looks at the path as it pertains to SQL Server service and that account being able to access it. The original statement was checking if the path existed using an external credential (if passed in, `$SourceCredential`). The SourceCredential is utilized to access that path under Windows Authentication.  Since `Test-DbaSqlPath` directly connects to the SQL Server instance that Source credential does not (or may not) have access to the given SQL Server instance, but regardless you are validating the path based on the rights of the SQL Server service account.

The fix for this is to utilize `Invoke-Command2` to actually ensure the `Test-Path` is run on the remote system, as well that command handles the credential. You can see this done on line 703 of this command:
```powershell
Invoke-Command2 -ComputerName $DestinationServerName -Credential $DestinationCredential -ScriptBlock 
```